### DISCUSSION TITLE
Stream Enhancement & Throughput Increase

### DISCUSSION THREAD

**Discussion by AdiAbuAli** [mrlt8/docker-wyze-bridge#249] [Ideas]
Hi,
I was suffering from lagging streams, low throughput, and low FPS, till I disabled WMM QoS in my router.
Unfortunately, Wyze cams doesnt support WMM, and normally this option cause problem in multimedia as per other users experiences...after I disabled it on both 2 and 5Ghz (wyze use 2Ghz)...throughput jumped more than 2x -3x !!

Below is a description of this feature (IEEE optional standard) in most of the routers:

Wi-Fi Multimedia Quality of Service (WMM QoS)
WMM QoS prioritizes WiFi data packets from different applications based on four access
categories: voice, video, best effort, and background. For an application to receive the
benefits of WMM QoS, WMM must be enabled for both it and the client running that
application. Legacy applications that do not support WMM and applications that do
not require QoS are assigned to the best effort category, which receives a lower priority
than voice and video.

**Comment by mrlt8** [comment]
Probably best to also disable non-standard wifi "enhancements" like [airtime fairness](https://support.wyze.com/hc/en-us/articles/360031479211-ASUS-Routers-and-Airtime-Fairness) on the 2.4 Ghz and move all non-IoT traffic to 5Ghz.

**Comment by AdiAbuAli** [comment]
It depends on the manufacture and technology...
For me its Orbi mesh wifi...so neither I have airfairness neither you can control to choose 2g or 5g....but good for others to pay attention on router configurations

Thanks,
________________________________
